### DISCUSSION TITLE
我想在阿里云上部署，有些rss需要外网，应该怎么做

### DISCUSSION THREAD

**Discussion by ishuliang** [DIYgod/RSSHub#15757] [Q&A]
### 这是一个什么样的功能？

部署功能

### 这个功能可以解决什么问题？

之后看就不需要挂梯子

### 这不是重复的功能请求和 RSS 提案

- [X] 我已经搜索了 [现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求及不是 [新的 RSS 提案](https://github.com/DIYgod/RSSHub/issues/new?assignees=&labels=RSS+proposal&template=rss_request_zh.yml)。

**Comment by TonyRL** [comment]
#12014
